autogeneration into function doc (#16242)
kyma-project/kyma: docs/05-technical-reference/00-custom-resources/svls-01-function.md

This table lists all the possible parameters of a given resource together with their descriptions:

> <!-- Part of the table bellow is generated automatically, please don't touch them. When you want to change description, do it in components/function-controller/pkg/apis/serverless/v1alpha2/function_types.go, then regenerate using "make manifests" in components/function-controller. -->

Reviewer: ```suggestion
<!-- Part of the following table is generated automatically, don't alter it. When you want to change the description, do it in components/function-controller/pkg/apis/serverless/v1alpha2/function_types.go, then regenerate using "make manifests" in components/function-controller. -->
```

> <!-- Some special tags (html comments) are at the end of lines due to markdown requirements. -->

Reviewer: ```suggestion
<!-- Some special tags (HTML comments) are at the end of the lines due to the Markdown requirements. -->
```

> <!-- The content between "FUNCTION-SPEC-START" and "FUNCTION-SPEC-END" will be replaced but rows with "KEEP-THIS" at the end will be preserved. -->

Reviewer: ```suggestion
<!-- The content between "FUNCTION-SPEC-START" and "FUNCTION-SPEC-END" will be replaced, but the rows with "KEEP-THIS" at the end will be preserved. -->
```

> <!-- All comments in this file that contain SKIP-ELEMENT and the name of the element will skip the generation of that element. Similarly with SKIP-WITH-ANCESTORS comments, which skip elements along with their descendants. -->

Reviewer: ```suggestion
<!-- All comments in this file that contain SKIP-ELEMENT and the name of the element will skip the generation of this element. Similarly, SKIP-WITH-ANCESTORS comments skip elements along with their descendants. -->
```

> <!-- Words in descriptions that are surrounded by asterisks will be replaced with parameter names with full path and surrounded by double asterisks -->

Author: Look on this

> <!-- Words in descriptions that are surrounded by asterisks will be replaced with parameter names with full path and surrounded by double asterisks -->

Reviewer: ```suggestion
<!-- Words in the descriptions that are surrounded by asterisks will be replaced with parameter names with full path and surrounded by double asterisks -->
```
